experiments/functional/jiva-volume-resize/daemonset.j2

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: jiva-resize
+spec:
+  selector:
+    matchLabels:
+      app: jiva-resize
+  template:
+    metadata:
+      labels:
+        app: jiva-resize
+    spec:
+      containers:
+      - name: jiva-resize
+        image: somesh2905/jiva-resize
```

Comment:
Image name?

Reply:
Its a customised image, which will be used in daemonset.